package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "ftw.sliderblock",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "git@github.com:4teamwork/ftw.solr.git",
+  "author": "4teamwork AG <info@4teamwork.ch>",
+  "license": "GPL2",
```

Comment:
``` "private": true ``` is missing https://devdocs.4teamwork.ch/plone/javascript/ftw-resource-bundling-plone5/#bundling-with-yarn-and-friends-outside-plone

Reply:
Done